### DISCUSSION TITLE
Release on teleport.dev with an latest tag

### DISCUSSION THREAD

**Discussion by maddeye** [gravitational/teleport#22216] [Ideas]
Hello there,

Lately i wanted to deploy Teleport with [terraform](https://www.terraform.io/) and [cloud-init](https://cloud-init.io/). One problem i encountered with this, is that i want the latest version of teleport deployed without constantly updating my terraform scripts.

Would it be possible to release new releases to
`https://apt.releases.teleport.dev/ubuntu $RELEASE stable/v{MAJOR_VERSION}`
and
`https://apt.releases.teleport.dev/ubuntu $RELEASE stable/latest` ?

This would benefit every automatic deployment in the future

**Comment by webvictim** [comment]
This is a similar request to https://github.com/gravitational/teleport/issues/2777
